Marlin/src/lcd/extui/lib/mks_ui/pic_manager.cpp

```diff
@@ -465,7 +488,7 @@ uint8_t public_buf[512];
         if (card.longFilename[0] == 0) continue;
         if (card.longFilename[0] == '.') continue;
 
-        uint8_t a = arrayFindStr(assets, COUNT(assets), card.longFilename);
+        int8_t a = arrayFindStr(assets, COUNT(assets), card.longFilename);
```

Comment:
Changed this to an `int8_t` because `a >= 0` was allowing the error value of -1 through.